.envrc

```diff
@@ -19,4 +19,6 @@ fi
 
 # Podman compatibility
 # NOTE: Exits direnv if user is using docker.
-source_env_if_exists .env.podman
+source_env_if_exists .envrc.podman
+
+source_env_if_exists .envrc.kube
```

Comment:
We had an issue once where envs weren't loaded after .envrc.podman. See the comment above.

Probably makes sense to fix the podman script to not exit envrc.

Reply:
Yup, changed to `return` instead.

Working on generalization [here](https://github.com/island-is/island.is/pull/16458)